keyboards/planck/keymaps/mwpeterson/keymap.c

```diff
@@ -81,15 +92,15 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    *      window    ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
    *    switcher    │     │  1  │  2  │  3  │  4  │  5  │  6  │  7  │  8  │  9  │  0  │     │
    *                ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
-   *                │     │  -  │  +  │  `  │  |  │  :  │     │     │  ,  │  .  │  \  │     │
+   *                │     │  -  │  +  │  `  │  |  │  :  │     │  ‽  │  ,  │  .  │  \  │     │
    *                ├─────┼─────┼─────┼─────┼─────┼─────┴─────┼─────┼─────┼─────┼─────┼─────┤
    *                │     │     │     │     │     │ Backspace │     │     │     │     │     │
    *                └─────┴─────┴─────┴─────┴─────┴───────────┴─────┴─────┴─────┴─────┴─────┘
    */
   [LOWER_LAYER] = LAYOUT_planck_grid(
     KC_GRV,   KC_F1,   KC_F2,    KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  S(KC_3),
     _______,  KC_1,    KC_2,     KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______,
-    _______,  KC_MINS, KC_PLUS,  KC_GRV,  KC_PIPE, KC_COLN, XXXXXXX, XXXXXXX, KC_COMM, KC_DOT,  KC_BSLS, _______,
+    _______,  KC_MINS, KC_PLUS,  KC_GRV,  KC_PIPE, KC_COLN, XXXXXXX, X(IBANG), KC_COMM, KC_DOT,  KC_BSLS, _______,
```

Comment:
```suggestion
    _______,  KC_MINS, KC_PLUS,  KC_GRV,  KC_PIPE, KC_COLN, XXXXXXX, X(IBANG),KC_COMM, KC_DOT,  KC_BSLS, _______,
```
Don't ruin the nice alignment with the surrounding lines 😛